docs/js/form.md

```diff
@@ -8,20 +8,27 @@ path: form
 ---
 
 # Form
+<p class="lead">
+  The form component is built as a jQuery plugin responsible for handling the state of input elements.
+</p>
 
 ### How to use
-As in right now, the `form` plugin is responsible to handle the `.active` state of the `.field`:
-
+By passing an object as a parameter to the `form` function, you can define the `event` that will trigger the component's action, as well as list the `selectors` attached to that event.
 ```js
-// accepted options / default values:
+// accepted options
 var options = {
   events: 'change',
   selectors: '.input, select, .select, .textarea'
 };
 
 $('[data-form]').form(options);
 ```
+<p class="notification notification-warning">
+  Notice that the strings used in the example above are the default values.
+</p>
 
-The constructor will loop through each `selector` found on the element and check if it has either a `value` or `placeholder`, then add/remove the `.active` class on the `.field` molecule. If the `selector` does not have a `.field` and parent, the verification will be skipped, and no classes will be added.
+With that, whenever a `change` event occurs in one of the selectors listed, the component will toggle the `.active` class, if needed.
 
-The provided event listener will do the same verification process.
+### How it works
+The component constructor will loop through each `selector` found on the element and check if it has either a `value` or a `placeholder`, then add or remove the `.active` class on the `.field` molecule. If the `selector` does not have a `.field` as a parent, the verification will be skipped, and no classes will be added.
```

Comment:
In this case I think I can't use `from` because I'm using the verbs `add` and `remove`, so it would have to be something that relates to both of them. The preposition `from` would make sense only for the `remove` action.